drivers/doc.txt

```diff
@@ -43,3 +43,10 @@
  * @ingroup     drivers
  * @brief       Drivers for storage devices
  */
+
+/**
+ * @defgroup    drivers_can CAN Drivers
+ * @ingroup     drivers
+ * @ingroup     can
```

Comment:
(don't know where this is defined, but) we don't have a directory `can` in RIOTBASE, so NACK for the name alone. Second, why do we need it? Third, why ist there a whole subgroup under drivers, just for CAN alone?